.ci/install_runtime.sh

```diff
@@ -63,6 +66,19 @@ build_install_shim_v2(){
 
 build_install_shim_v2
 
+build_install_agent_ctl(){
+	if [ ! -d "$agent_ctl_path" ]; then
```

Comment:
You should use the `clone_katacontainers_repo()`  method from `.ci/lib.sh` (already sourced). Then use the already exported `katacontainers_repo_dir` variable to reference the path to the kata-containers repo.